src/Data/Auth.Persistence/Entities/Account.cs

```diff
@@ -1,19 +1,57 @@
-﻿using Core.Persistence;
+﻿using System.ComponentModel;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
 
 namespace QuantumCore.Auth.Persistence.Entities;
 
-public class Account : BaseModel
+[Table("accounts")]
+public class Account
 {
-    public string Username { get; set; } = "";
-    public string Password { get; set; } = "";
+    public Guid Id { get; init; }
+    [StringLength(30)] public required string Username { get; init; } = "";
+    [StringLength(60)] public required string Password { get; set; } = "";
+    [StringLength(100)] public required string Email { get; init; } = "";
+    [DefaultValue(1)] public required short Status { get; init; }
```

Comment:
An Enum with actual typed status would make the usage of this property more clearer while also removing some possible magic numbers in the code.

Reply:
Actually the status is not a magic number but an ID to an entity. Not very intuitive but it's legacy